CMSgov/mdct-mcr
# Britt and Nicks Most Excellent MCPAR Dashboard Integration Tests

## Description


Fixes [MDCT-2074](https://qmacbis.atlassian.net/browse/MDCT-2074) and its children

### How to test

Make sure you checkout the Gherkin/Cucumber section in the Readme that Thomas wrote up! It's a great intro into how everything works and how to set it up. Once you've followed it, you can simply run (Make sure you swap passwordhere for the actual passwords!)

`CYPRESS_ADMIN_USER_PASSWORD='passwordhere' CYPRESS_STATE_USER_PASSWORD='passwordhere' yarn test`

That will bring up the cypress menu. Once everything spins up, you can click start integration test and then click all any test you'd like to see run. I'd recommend Dashboard and Form as that's what we're really adapting on this PR.

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
